src/test/java/com/google/firebase/snippets/FirebaseAuthSnippets.java

```diff
@@ -151,13 +151,13 @@ public static void updateUser(String uid) throws FirebaseAuthException {
     // [END update_user]
   }
 
-  public static void setCustomUserClaims(
+  public static void setCustomUserClaims(TenantAwareFirebaseAuth tenantAuth,
```

Comment:
I think it would be better to add a new function like "setCustomUserClaimsTenant" in https://github.com/firebase/firebase-admin-java/blob/1eec932b27276ddf1d6ab491a08edb68a3036170/src/test/java/com/google/firebase/snippets/FirebaseAuthSnippets.java#L1152

Reply:
Agreed. 
I reverted the changes made in `setCustomUsersClaims` and created a new function `setCustomUserClaimsTenant` that sets custom claims on a user for a specific tenant.